src/Gateway.php

```diff
@@ -170,6 +166,13 @@ public function setUsername(string $username): self
         return $this;
     }
 
+    public function setTestMode($testMode): self
+    {
+        $this->setParameter('testMode', $testMode);
```

Comment:
I'd use `parent::setTestMode` here, so it doesn't need to be kept in sync with any changes to the parent implementation.